cove_360/fixtures/SOURCES

```diff
@@ -1,12 +1,20 @@
+WellcomeTrust-grants_2_grants_cp1252.csv
```

Comment:
I would prefer to not have any organisation names in filenames at all, especially given the level of editing that we've done to the data to make it break. It's right and proper that the original source be credited in documentation but I think that it's unfair to Wellcome to have their name associated with this testing data every time anyone looks at the tests - especially as CoVE is rapidly becoming more of a community project.